docs/_data/sidebars/left-navigation-sidebar.yml

```diff
@@ -120,10 +124,6 @@ entries:
       url: /components/form.html
       output: web
 
-    - title: Identifier
-      url: /components/identifier.html
-      output: web
-
```

Comment:
I think we should keep it for at least 1 release marked as deprecated

Reply:
shall we bring it back?